presto-docs/src/main/sphinx/release/release-0.275.rst

```diff
@@ -0,0 +1,27 @@
+=============
+Release 0.275
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix Disaggregated Coordinator to correctly identify running queries as leaked.
+* Add :func:`trimmed_mean` for :ref:`tdigest <tdigest_type>`.
+* Add session property ``query_max_output_positions`` and configuration property ``query.max-output-positions`` to control how many rows a query can output. The query might end up returning more rows than the limit as the check is asynchronous.
+* Add support for setting session properties for presto on spark.
```

Comment:
This is spark change

Reply:
removed

Reply:
no, not remove; it's to add

Spark Changes
______________
* ...

Reply:
Also, IIRC there were raft and iceberg changes. Why do they all get removed?

Reply:
For Ratis change, spoke with Tim, feature is not complete, no user impact. they decided to take it off release notes.
For Iceberg, you mentioned there is not much user-facing impact since it is adding fields in system table, so removed that one too.

Reply:
OK, let's only add spark changes then

Reply:
updated.